src/amchart/CommonSerial.js

```diff
@@ -31,6 +31,8 @@
     CommonSerial.prototype.constructor = CommonSerial;
     CommonSerial.prototype._class += " amchart_CommonSerial";
 
+    CommonSerial.prototype.publish("backwardsCompatible", true, "boolean", "Allow use of old publish parameters");
+
```

Comment:
Remove - for the 1.x releases this should always be true and should not be an option.